rebalance/rebalance.py

```diff
@@ -684,6 +684,36 @@ def health_score(liquidity):
     return score * coefficient
 
 
+def get_avg_forward_fees(plugin: Plugin, intervals):
+    now = time.time()
+    max_interval = max(intervals)
+    total = [0] * len(intervals)
+    fees = [0] * len(intervals)
+    res = [0] * len(intervals)
+    # TODO: next commented out line dows not work. pyln binding bug?
+    #all_forwards = plugin.rpc.listforwards(status="settled")["forwards"]
+    all_forwards = list(filter(lambda fwd: fwd.get("status") == "settled",
+                               plugin.rpc.listforwards()["forwards"]))
```

Comment:
Maybe `fwd.get("status") == "settled" and fwd.get("resolved_time", 0) + max_interval * 60 * 60 * 24 > now`, or something like that?
This way the following code may be cleaner, because of no need for `sort` and `break` in the loop.

Reply:
hm, yes but we actually must fix pyln to accept the "status" when binding. When this is done the line 703 that checks for "resolved_time" later on must keep in place anyway.

Reply:
I also checked, when status is resolved, there is always a "resolved_time" in my database

Reply:
> I also checked, when status is resolved, there is always a "resolved_time" in my database

Not always. Can you check really old forwards, before v0.7.0?
My database is continuous since 2018 July. 😎

Reply:
:D will add this. do you have an idea why `plugin.rpc.listforwards(status="settled")` does not work?